MidCamp/midcamp
# Theme switching

# Description

Re-enables the old Hatter theme and sets it as the site default.  Introduces the [Switch Page Theme](https://www.drupal.org/project/switch_page_theme) module to leave the green "O'MidCamp" theme enabled on`/2020*` pages only.

# To test

- (Optional) Build locally
  - Install dependencies with `composer install`
  - Import configuration with `drush cim -y`
  - Follow the test instructions below with your local (using `midcamp.org.docker.amazee.io` URL)
- Visit the homepage (https://nginx.feature-theme-switching.midcamp-org.us2.amazee.io/) and observe the old theme is enabled.
- Visit 2021 event year pages like [`/2021/news`](https://nginx.feature-theme-switching.midcamp-org.us2.amazee.io/2021/news) or [`/2021/about`](https://nginx.feature-theme-switching.midcamp-org.us2.amazee.io/2021/about).  Observe the old theme is enabled.
- Visit 2020 event year pages like [`/2020/organizers`](https://nginx.feature-theme-switching.midcamp-org.us2.amazee.io/2020/organizers) or [`/2020/about`](https://nginx.feature-theme-switching.midcamp-org.us2.amazee.io/2020/about).  Observe the green theme is enabled.
